src/components/CalendarDay.js

```diff
@@ -10,7 +10,9 @@ const CalendarDay = ({day, isToday, events}) => {
     });
 
     return (
-        <div className={dayClasses}>
+      <div 
+        onClick={onClick.bind(null, 'day', day)}
```

Comment:
Nevermind I see your above comment on this in the PR description. I'd still argue passing the "day" string is completely useless.